packages/jaeger-ui/src/components/TracePage/index.test.js

```diff
@@ -132,6 +135,32 @@ describe('<TracePage>', () => {
     });
   });
 
+  describe('focusUiFindMatches', () => {
+    let scrollToFirstVisibleSpanSpy;
+
+    beforeEach(() => {
+      scrollToFirstVisibleSpanSpy = jest.spyOn(wrapper.instance()._scrollManager, 'scrollToFirstVisibleSpan');
+      scrollToFirstVisibleSpanSpy.mockReset();
+      defaultProps.focusUiFindMatches.mockReset();
+    });
+
+    it('calls _scrollManager.scrollToFirstVisibleSpan and props.focusUiFindMatches with props.trace.data and uiFind when props.trace.data is present', () => {
```

Comment:
Is the call to `_scrollManager.scrollToFirstVisibleSpan` necessary?

Reply:
nope, removed :+1: